### PR TITLE
Tab: Fix focus trap

### PR DESCRIPTION
**Pull request by jamesducky**
### Summary of changes included in this PR

- Removes focus trap by setting blur on enter/tab keyboard input replicating accessible behaviour [seen here](https://lion-web.netlify.app/components/tabs/overview/#tabs-overview)
- Adds some missing returns statements and typescript 
- Removes a missing import bug from react app

### Reviewer checklist:

- Tests for
  - accessibility
  - rendered HTML (spec)
- Storybook stories for all variants
- JSDoc documentation for component
- Update Example app (React, ...)

### CODE REVIEW

File: examples/react-app/src/index.tsx [hunk 1]
```diff
@@ -131,7 +131,6 @@ ReactDOM.render(
       <PlmgSvgIcon icon={'home'} size={'6em'} />
       PlmgSvgIcon home
       <br />
-      <Dropdown />
```
Comment (TanguyDucky) [inline examples/react-app/src/index.tsx:134]: does not seem to have anything to do with this PR?

Comment (jamesducky) [reply examples/react-app/src/index.tsx:134]: This is true. I was being lazy. I'll make a separate one :)

File: packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx [hunk 2]
```diff
@@ -122,4 +126,13 @@ export class Tabs {
       this.onChange.emit({ tabId: tabIndex });
     }
   }
+
+  private blurTabButton(tabIndex: number) {
+    const buttons = document.getElementsByClassName(
+      'plmg-tab-button'
+    ) as HTMLCollectionOf<HTMLButtonElement>;
+    if (document.activeElement === buttons.item(tabIndex)) {
```
Comment (TanguyDucky) [inline packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx:134]: I feel like this logic assume there is only one Tabs on the page. What if they are multiples?

File: packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx [hunk 3]
```diff
@@ -101,11 +105,11 @@ export class Tabs {
     );
   }
 
-  private hasIcon(icon) {
+  private hasIcon(icon: string) {
     return icon && (icon as string) !== '';
   }
```
Comment (JohannesAnd) [inline packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx:110]: Not a full review. but I thought I'd comment on this as it caught my eye:

- I'd probably do `icon?: string` as it is optional
- An empty string is a falsy value, so the last part of this check will never be false

So, unless I am mistaken by some curiosity of Stencil or web components, I would have done this:

```javascript
private hasIcon(icon?: string) {
  return Boolean(icon);
}
```

Comment (jamesducky) [reply packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx:110]: Ah. That makes sense. thankyou!

File: packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx (hunk 2 above)
Comment (jamesducky) [reply packages/component-library/src/components/plmg-tabs/plmg-tabs.tsx:134]: Now restricted to component scope.

File: examples/react-app/src/index.tsx (hunk 1 above)
Comment (jamesducky) [reply examples/react-app/src/index.tsx]: Reverted. Moved bug fix to [this PR](https://github.com/DuckyTeam/Plumage/pull/77)

**Comment by jamesducky** [comment]
@TanguyDucky. Made changes. 
- Query selector restricted to component scope.
- Typescript changes as suggested by @JohannesAnd

**Review by TanguyDucky — APPROVED** [review]

**Pull request merged**